test/controllers/ApplicationControllerSpec.scala

```diff
@@ -18,23 +18,17 @@ package controllers
 
 import audit.Auditable
 import config.ApplicationConfig
-import models.FeedBack
 import org.jsoup.Jsoup
 import org.scalatestplus.play.PlaySpec
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
 import play.api.i18n.{Lang, Messages, MessagesApi}
-import play.api.libs.json.Json
-import play.api.mvc.{Headers, MessagesControllerComponents}
+import play.api.mvc.MessagesControllerComponents
 import play.api.test.Helpers._
 import play.api.test.{FakeRequest, Injecting}
 
-
-import java.util.UUID
-
 class ApplicationControllerSpec extends PlaySpec with GuiceOneServerPerSuite with Injecting {
   val service = "ATED"
   val injectedViewInstanceUnauthorised = inject[views.html.unauthorised]
-  val injectedViewInstanceFeedBack = inject[views.html.feedback]
   val injectedViewInstanceThankYou = inject[views.html.feedbackThankYou]
```

Comment:
injectedViewInstanceThankYou should not be required and consequently the html.view can be removed as its all part of the the same feedback controller detail being removed ?

Reply:
Removed